### ISSUE TITLE
Text formatting does not specify culture

### ISSUE THREAD

**Issue by tom-englert** [Microsoft/CodeContracts#149]
String formatting in the analyzer does not specify a culture, therefore some tests fail on systems with non-English culture with e.g. "expected: 2.0, actual: 2,0"

I have added a workaround in ```ClousotMain.Analyze()``` by setting the main threads culture to invariant. However this is very bad practice, and should be fix in long term.

**Comment by SergeyTeplyakov** [comment]
Yep, totally agree. It should be automated way to switch to culture agnostic formatting. I think R# could do this.

**Comment by tom-englert** [comment]
Currently it's hard to use R# because the formatting of the code is not yet cleaned up. 
Once we have the code properly formatted, we could add a R#.sln.DotSettings file so everyone is using the same style.

**Comment by SergeyTeplyakov** [comment]
I meant that even in this case it is possible to run only one fix across all projects. I'll try to do this.

**Comment by sharwell** [comment]
> Once we have the code properly formatted, we could add a R#.sln.DotSettings file so everyone is using the same style.

I'm not using ReSharper. I'm planning to add a proposal that we use StyleCopAnalyzers for style checking. It has no dependencies other than Roslyn, is free for everyone, is extremely fast, and we've implemented the ability to fix most violations with 3 clicks or less.

**Comment by tom-englert** [comment]
I don't mind using StyleCopAnalyzers, I think it does not contradict with making the live of R# users easier :smile:
